samples/simple/sample_test_run.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2019 Intel Corporation.  All rights reserved.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+
+import argparse
+import shlex
+import subprocess
+import sys
+import time
+import traceback
+
+
+def start_server(cwd):
+    """
+    Startup the 'simple' process works in TCP server mode
+    """
+    app_server = subprocess.Popen(shlex.split("./simple -s "), cwd=cwd)
+    return app_server
+
+
+def query_installed_application(cwd):
+    """
+    Query all installed applications
+    """
+    query_process = subprocess.run(
+        shlex.split("./host_tool -q"), cwd=cwd, check=False, capture_output=True
+    )
+    assert query_process.returncode == 69
+    return query_process.returncode, query_process.stdout
+
+
+def install_wasm_application(wasm_name, wasm_file, cwd):
+    """
+    Install the request handler wasm application
+    """
+    install_process = subprocess.run(
+        shlex.split(f"./host_tool -i {wasm_name} -f {wasm_file}"),
+        cwd=cwd,
+        check=False,
+        capture_output=True,
+    )
+    assert install_process.returncode == 65
+    return install_process.returncode, install_process.stdout
+
+
+def send_get_to_wasm_application(wasm_name, url, cwd):
+    """
+    send a request (GET) from host to an applicaton
+    """
+    query_process = subprocess.run(
+        shlex.split(f"./host_tool -r /app/{wasm_name}{url} -A GET"),
+        cwd=cwd,
+        check=False,
+        capture_output=True,
+    )
+    assert query_process.returncode == 69
+    return query_process.returncode, query_process.stdout
+
+
+def main():
+    """
+    GO!GO!!GO!!!
+    """
+    parser = argparse.ArgumentParser(description="run the sample and examine outputs")
+    parser.add_argument("working_directory", type=str)
+    args = parser.parse_args()
+
+    ret = 1
+    app_server = None
+    try:
+        app_server = start_server(args.working_directory)
+
+        # wait for a second
+        time.sleep(1)
+
+        print("--> Query all installed applications...")
+        query_installed_application(args.working_directory)
+        print("--> Install a Wasm application named request_handler...")
+        install_wasm_application(
+            "request_handler",
+            "./wasm-apps/request_handler.wasm",
+            args.working_directory,
+        )
```

Comment:
only install one wasm app, how about install more:
timer.wasm
event_publisher
event_subscriber    (subscriber can then communicate with publisher)
Then uninstall them, and install
request_handler
request_sender